src/cart/crt_context.c

```diff
@@ -2147,6 +2163,7 @@ get_quota_resource(crt_context_t crt_ctx, crt_quota_type_t quota)
 	} else {
 		D_DEBUG(DB_TRACE, "Quota limit (%d) reached for quota_type=%d\n",
 			ctx->cc_quotas.limit[quota], quota);
+		d_tm_inc_counter(ctx->cc_quotas.rpc_quota_exceeded, 1);
```

Comment:
hmm.. how is this different from line 1388? this place returns rc, line 1388 is where it is used, but we do 2 different metrics which is confusing

Reply:
waitq_depth (guage) tracks the depth of the queue whereas this tracks how many times total we exceed the limit (counter).   Perhaps we could move them next to each other or we could remove one.   @mjmac added these originally.